.devcontainer/compose.yml

```diff
@@ -15,6 +15,6 @@ services:
     entrypoint: mkdocs
     command: serve --dev-addr "0.0.0.0:8001"
     ports:
-      - "8001:8001"
+      - 8001:8001
```

Comment:
this is just me being neurotic and preferring numbers since thats what they show in the docker documentation.